docs/user-guide.md

```diff
@@ -434,29 +577,15 @@ TODO
 
 ## PageRank
 
-<div class="codetabs">
-
-<div data-lang="scala"  markdown="1">
-{% highlight scala %}
-import org.graphframes.examples
-val g: GraphFrame = examples.Graphs.friends  // get example graph
-
-TODO
-{% endhighlight %}
-</div>
-
-<div data-lang="python"  markdown="1">
-{% highlight python %}
-from graphframes.examples import Graphs
-g = Graphs(sqlContext).friends()  # Get example graph
-
-TODO
-{% endhighlight %}
-</div>
+There are two implementations of PageRank.
 
-</div>
+* The first implementation uses the standalone [[GraphFrame]] interface and runs PageRank
+ for a fixed number of iterations.  This can be run by setting `numIter`.
+* The second implementation uses the `org.apache.spark.graphx.Pregel` interface and runs PageRank
```

Comment:
What happens if we set both? I think this is implementation details. Maybe it is not necessary to mention that there are two implementations in the user guide.